asreview/webapp/start_flask.py

```diff
@@ -158,4 +170,9 @@ def _internal_open_webbrowser():
         seed=args.seed
     )
     app.config['PROPAGATE_EXCEPTIONS'] = False
-    app.run(host=host, port=port)
+
+    # use WSGI server if use_gevent flag is present
+    if args.use_gevent:
+        WSGIServer((host, port), app).serve_forever()
+    else:
+        app.run(host=host, port=port)
```

Comment:
A general question, should we use the gevent server by default? The Flask server might be relevant for developers only, isn't it? Gevant becomes a required dependency in this PR, so that should not be an issue. I'm wondering what your opinion is on this.